config/routes.rb

```diff
@@ -493,6 +485,82 @@
   lookup_user
 ].freeze
 
+# redirect deleted MO controller's actions to equivalent actions in the
+# equivalent normalized controller
+# Examples:
+#  redirect_old_crud_actions(old_controller: "article")
+#  redirect_old_crud_actions(old_controller: "herbarium")
+#  redirect_old_crud_actions(old_controller: "glossary",
+#                            new_controller: "glossary_terms",
+#                            actions: [:create, :edit, :show])
+#
+def redirect_old_crud_actions(
+  old_controller: "",
+  new_controller: old_controller&.pluralize,
+  model: new_controller&.singularize,
+  actions: [:create, :destroy, :edit, :index, :show]
+)
+
+  actions.each do |action|
+    send("redirect_#{action}", old_controller, new_controller, model)
+  end
+end
+
+# rubocop:disable Style/FormatStringToken
+# For consistency with Rails Guide "Rails Routing from the Outside In"
+# (And I don't know if RuboCop's suggestion works here)
+def redirect_create(old_controller, new_controller, model)
```

Comment:
This isn't quite what I was going for.  I'm working on it and I'll have a more concrete example shortly.